data/builtin_build_context/install_gpu.sh

```diff
@@ -60,22 +60,29 @@ main() {
   mount --bind "${NVIDIA_INSTALL_DIR_HOST}" "${NVIDIA_INSTALL_DIR_HOST}"
   mount -o remount,exec "${NVIDIA_INSTALL_DIR_HOST}"
   pull_installer
-  docker run \
+  if ! docker run \
```

Comment:
nit: How about putting the `docker run` command in a variable and using that in the "if" condition? I think it will be easy to follow the if condition in that case.

Reply:
Done